content/500-platform/20-concepts/10-workspaces.mdx

```diff
@@ -17,4 +17,4 @@ Workspaces provide a unified space for team collaboration on projects. Manage me
 
 For your Workspace, you can choose a subscription plan that aligns closely with your requirements. Every project within a Workspace adds to the collective usage, ensuring a holistic view of consumption.
 
-More details on our subscription plans will be available soon.
+All our paid plans are prorated, which means you're only billed for the duration you use a specific plan. For example, if you subscribe to our **Pro** plan for only 15 days in a 30-day month, you'll only be charged for half of the plan's price. Visit our [pricing page](https://www.prisma.io/pricing) for more details.
```

Comment:
Is prorated supposed to be pro-rated? I honestly don't know. Could you just make sure it matches how we use it in other places?

Reply:
Prorated should be the way to spell this. The pricing page doesn't mention this [too](https://www.prisma.io/pricing).

<img width="1118" alt="image" src="https://github.com/prisma/docs/assets/64993082/b5d05534-fb6c-4706-a4a2-9eccb696af26">